src/App.js

```diff
@@ -1,20 +1,51 @@
 import React, { Component } from 'react'
-import Catalogue from './Catalogue'
+import NavBar from './Components/NavBar/NavBar'
+import SelectedMovie from './Components/SelectedMovie/SelectedMovie'
+import Catalogue from './Components/Catalogue/Catalogue'
 import movieData from './mockData'
 import './App.css';
 
 class App extends Component {
     constructor(){
        super()
        this.state = { 
-           movies: movieData.movies
+           movies: movieData.movies,
+           selectedMovie: 0
         }
     }
+
+    returnToHome = event => {
+      this.setState({
+        selectedMovie: 0
+      })
+    }
+
+    handleClick = event => {
+        this.setState({ 
+            selectedMovie: event.target
+        })
+    }
+
```

Comment:
perhaps eventually we can make these one function

Reply:
Also event.target should resolve to an id, insted of setting the state to the full target